### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by JSteunou** [pwnall/node-open#39]
Is this project still alive @pwnal? There are a couple of PR / issues that need to be handled. Maybe you can open this project to get some external help.

**Comment by IonicaBizau** [comment]
I'm also interested in this. :question:

**Comment by sindresorhus** [comment]
Check out [`opn`](https://github.com/sindresorhus/opn) for a maintained alternative.

**Comment by JSteunou** [comment]
ty @sindresorhus

**Comment by IonicaBizau** [comment]
@sindresorhus Indeed. Much better. The funny thing is that I found it when I hit the idea of creating a better npm module that does this named `opn`. :smile:
